index.html

```diff
@@ -168,6 +181,85 @@ <h2>Profesional Art Printing Data</h2>
             </hrx>    
             <button type="button">See Project</button>
           </hrx>
+
+          <hrx class="work-cards-desk img-placeholder2">              
+            <button id = "button-width" type="button">See Project</button>
+          </hrx>
+
+          <hrx class="work-cards-desk img-placeholder4">
+            <h2>Data Dashboard <br> Healthcare</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+            
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
```

Comment:
- [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins). please use the same color for text inside the buttons and also remove the white border
- [ ] Please implement for all the buttons
### Your design
![your](https://user-images.githubusercontent.com/81727988/199896232-a2bf4ddd-627c-42b2-a330-fa8444e9b160.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/199896248-6229091c-ecfc-4a24-aaae-59e26effe023.png)